rfcs/rfc-009-staking.md

```diff
@@ -0,0 +1,59 @@
+Initial proof-of-stake (i.e. using in-protocol asset, SOL, to provide secure consensus) design ideas outlined here. Solana will implement a proof of stake reward/security scheme for node validators on the network. The purpose is threefold:
+
+- Align validator's incentives with that of the greater network through skin-in-the-game deposits at risk
+- Avoid 'nothing at stake' branch voting issues by implementing slashing rules aimed at promoting branch convergence
+- Provide an avenue for validator rewards provided as a function of validator participation in the network.
+- ?network security?
+
+While many of the details of the specific implementation are currently under consideration and are expected to come into focus through specific modeling studies and parameter exploration on the Solana testnet, we outline here our current thinking on the main components of the PoS system. Much of this thinking is based on the current status of Casper FFG, with optimizations and specific attributes to be modified as is allowed by Solana's PoH blockchain data structure.
+
+### General Overview
+Solana's validation network design is based on a rotating, stake-weight randomly selected leader broadcasting network transactions in a Proof of History data structure to validating nodes. These nodes, upon receiving the leader's broadcast, have the opportunity to vote on the current state and PoH height by signing a transaction into the PoH stream.
+
+To become a Solana validator, a network node must deposit/lock-up some amount of SOL in a contract. This SOL will not be accessible for a specific time period. The precise duration of the staking lockup period have not been determined, however we can consider three phases of this time:
```

Comment:
"have not" -> "has not"

Reply:
Also, "however we can consider three phases of this time" can be a separate sentence.  It'd also be helpful to clarify the purpose of breaking up the the time period into three phases before describing them.